pyvista/core/pointset.py

```diff
@@ -795,6 +794,61 @@ def offset(self):
         else:  # this is no longer used in >= VTK9
             return _vtk.vtk_to_numpy(self.GetCellLocationsArray())
 
+    def cast_to_explicit_structured_grid(self):
+        """Cast to an explicit structured grid.
+
+        Returns
+        -------
+        ExplicitStructuredGrid
+            An explicit structured grid.
+
+        Raises
+        ------
+        TypeError
+            If the unstructured grid doesn't have the ``'BLOCK_I'``, ``'BLOCK_J'`` and
+            ``'BLOCK_K'`` cells arrays.
+
+        See Also
+        --------
+        ExplicitStructuredGrid.cast_to_unstructured_grid :
+            Cast an explicit structured grid to an unstructured grid.
+
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>>
+        >>> grid = examples.load_explicit_structured()  # doctest: +SKIP
+        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP
+        >>>
+        >>> grid.hide_cells(range(80, 120))  # doctest: +SKIP
+        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP
+        >>>
+        >>> grid = grid.cast_to_unstructured_grid()  # doctest: +SKIP
+        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP
+        >>>
+        >>> grid = grid.cast_to_explicit_structured_grid()  # doctest: +SKIP
+        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP
```

Comment:
```suggestion
        >>> from pyvista import examples
        >>> grid = examples.load_explicit_structured()  # doctest: +SKIP
        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP

        >>> grid.hide_cells(range(80, 120))  # doctest: +SKIP
        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP

        >>> grid = grid.cast_to_unstructured_grid()  # doctest: +SKIP
        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP

        >>> grid = grid.cast_to_explicit_structured_grid()  # doctest: +SKIP
        >>> grid.plot(color='w', show_edges=True, show_bounds=True)  # doctest: +SKIP
```

Our sphinx docs will split these up into separate examples and it will look nicer when it renders.

Reply:
You're right! I will consider your suggestion for my next codes. Thanks!